tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -52,10 +46,18 @@
 import io.netty.handler.logging.LogLevel;
 import io.netty.handler.logging.LoggingHandler;
 import io.netty.util.HashedWheelTimer;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+import io.netty.util.internal.logging.Slf4JLoggerFactory;
+import org.apache.log4j.BasicConfigurator;
+import org.apache.log4j.Level;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.slf4j.LoggerFactory;
```

Comment:
it is customary not to tie Java libraries to a particular logging implementation. Why is it not enough to just log to slf4j? The actual application then can bind to whichever logging framework best suits it.